docs/compatible_configs.md

```diff
@@ -62,7 +63,21 @@ The option `force_grid_wrap = 0` is just to tell isort to only wrap imports that
 the `line_length` limit.
 
 Finally, isort should be told to wrap imports when they surpass _Black_'s default limit
-of 88 characters via `line_length = 88`.
+of 88 characters via `line_length = 88` as well as
+`ensure_newline_before_comments = True` to ensure spacing import sections with comments
+works the same as with _Black_.
+
+---
+
+NOTE
+
+`ensure_newline_before_comments = True` only works since isort >= 5 but does not break
+previous ones so you can keep it if you are running previous versions. If only isort >=
+5 is used you can add `profile = black` instead of all the options since
+[profiles](https://timothycrosley.github.io/isort/docs/configuration/profiles/) are
+available and do the configuring for you.
+
+---
```

Comment:
I personally do not like the use of visible `<br>`s here since they feel more segregating than headers IMO. I recommend either using a level 4 header (e.g. `#### NOTE: with isort 5+`) or do something similar to the note at the top of this doucment. OR, you could change the sectioning with two sections, one for isort 4=< with a note regarding the mostly useless option, and the other section for isort 5+

Your call, but IMO, the current formatting looks a bit ugly right now.

Reply:
Since this a black repository I should not come up with new ideas for how to style things and adopted the style already present in the document.